app/model/exchange/sift/SiftAnswers.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model.exchange.sift
+
+import model.exchange.sift.SiftAnswersStatus.SiftAnswersStatus
+import play.api.libs.json._
+
+object SiftAnswersStatus extends Enumeration {
+  type SiftAnswersStatus = Value
+
+  val DRAFT, SUBMITTED = Value
```

Comment:
Two similar enums. I've seen others put enum `AllocationStatuses` to `model` package, to avoid duplication. Don't know what is the best practice here.